### DISCUSSION TITLE
Run command after ssh login

### DISCUSSION THREAD

**Discussion by carloslubas** [mRemoteNG/mRemoteNG#2583] [General]
Hello everybody!

I looked in the mRemoteNG documentation and also in the application and did not find an option to execute a command after logging in via ssh.
Is there this option?
Something similar to "External tools", but run after logging into the remote host or even the "Enable post-login commands" option in the "PuTTY Connection Manager" tool.

**Comment by Kvarkas** [comment]
If you look under **Miscellaneous** section you should see it, as example this one from my Linux box:
![image](https://github.com/mRemoteNG/mRemoteNG/assets/3611964/378ff315-3f8b-43d4-8a19-c3f89e297662)
I know interface a bit not obliquus but will work on that a bit later then refactoring UI

**Comment by carloslubas** [comment]
Sorry, I didn't find the "Miscellaneous" section, I'm using the portable stable version, 1.76.20.24669, on Windows 11 (I use Windows at work), is there a difference in version and because it's portable? Should I use another version?

**Reply by Kvarkas** [reply]
dont remember but perhaps we add this in new version, curent one 1.77.3 nb but that not for production only for testing, working to release stable one so you could use it for yours work.

P.S. Night Build version is working version but disabled saving settings to avoid damage for ones who want to test it and dont ruin they stable version at same time.

**Comment by carloslubas** [comment]
I'm going to install a version that has a "miscellaneous" section, I was using the portable version to test mRemoteNG, now that I've tested it and liked it, I'm going to install it, which version do you recommend?

**Comment by carloslubas** [comment]
I installed version NB 1.77.3, it's working fine for me.
Thanks for the excellent work!
Problem solved!
